src/Systmm.f90

```diff
@@ -182,116 +181,116 @@ SUBROUTINE SYSTMM(temp_file,param_file)
             T_0 = T_head(nr)
           else 
 !
-!     Interpolation at the downstream boundary
+!     Interpolation at the downstream boundary
 !
           if(nseg.eq.no_celm(nr)) npndx=3
-!
-          do ntrp=1,nterp(npndx)
-            npart=nseg+ntrp+ndltp(npndx)
-            xa(ntrp)=x_dist(nr,npart)
-            ta(ntrp)=temp(nr,npart,n1)
-          end do
-          x=x_part(ns)

-!
-!     Call the interpolation function
-!
+!
+          do ntrp=1,nterp(npndx)
+            npart=nseg+ntrp+ndltp(npndx)
+            xa(ntrp)=x_dist(nr,npart)
+            ta(ntrp)=temp(nr,npart,n1)
+          end do
+          x=x_part(ns)
+!
+!     Call the interpolation function
+!
           T_0=tntrp(xa,ta,x,nterp(npndx))
           end if
-!
-300 continue
+!
+300 continue
 350 continue
-!
          dt_calc=dt_part(ns)
-          nncell=segment_cell(nr,nstrt_elm(ns))
-!
-!    Set NCELL0 for purposes of tributary input
-!
-          ncell0=nncell
-          dt_total=dt_calc
-          do nm=no_dt(ns),1,-1
-            z=depth(nncell)
-            call energy(T_0,q_surf,nncell)
-            q_dot=(q_surf/(z*rfac))
-            T_0=T_0+q_dot*dt_calc
-            if(T_0.lt.0.0) T_0=0.0
-!
-!     Look for a tributary.
-!
-            Q1=Q_in(nncell)
-            ntribs=no_tribs(nncell)
-            if(ntribs.gt.0.and..not.DONE) then
-              do ntrb=1,ntribs
-                nr_trib=trib(nncell,ntrb)
-                if(Q_trib(nr_trib).gt.0.0) then
-                  Q2=Q1+Q_trib(nr_trib)
-                  T_0=(Q1*T_0+Q_trib(nr_trib)*T_trib(nr_trib))/Q2
+!          dt_calc=dt_part(ns)
+          nncell=segment_cell(nr,nstrt_elm(ns))
+!
+!    Set NCELL0 for purposes of tributary input
+!
+          ncell0=nncell
+          dt_total=dt_calc
+          do nm=no_dt(ns),1,-1
+            z=depth(nncell)
+            call energy(T_0,q_surf,nncell)
+            q_dot=(q_surf/(z*rfac))
+            T_0=T_0+q_dot*dt_calc
+            if(T_0.lt.0.0) T_0=0.0
+!
```

Comment:
Not sure what is going on. I think git is perhaps thrown off a bit by all the whitespace changes, which clutter this entire PR. The better way would have been to fix all the whitespace changes separately and then make the code changes.